package.json

```diff
@@ -16,7 +16,7 @@
   "author": "Traceur Authors",
   "license": "Apache License 2.0",
   "engines": {
-    "node": ">=0.10"
+    "node": ">=0.12"
```

Comment:
I don't want to investigate issues on arbitrary versions if the vast majority of users are on recent versions.